query_optimizer/tests/execution_generator/CMakeLists.txt

```diff
@@ -17,39 +17,39 @@ add_test(quickstep_queryoptimizer_tests_executiongenerator_create
          "../quickstep_queryoptimizer_tests_ExecutionGeneratorTest"
          "${CMAKE_CURRENT_SOURCE_DIR}/Create.test"
          "${CMAKE_CURRENT_BINARY_DIR}/Create.test"
-         "${CMAKE_CURRENT_BINARY_DIR}/")
+         "${CMAKE_CURRENT_BINARY_DIR}/Create/")
```

Comment:
When run concurrently, two tests can manipulate the same `.qsb` file. In this case, we found that it threw an exception in the [FileManager](https://github.com/pivotalsoftware/quickstep/blob/master/storage/FileManagerPosix.cpp#L89). We fixed this problem by having the tests create their `.qsb` files in separate directories.